documentation/src/test/java/example/extensions/HttpServerExtension.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2015-2024 the original author or authors.
```

Comment:
Looks like new file, so why 2015?

Reply:
The licence information was generated with `gradlew spotlessApply` script, as per the contribution guide.